i18n/en.pot

```diff
@@ -136,6 +139,27 @@ msgstr ""
 msgid "Generic"
 msgstr ""
 
+msgid "Show"
+msgstr ""
+
+msgid "per page"
+msgstr ""
+
+msgid "Viewing {{firstItem}}-{{lastItem}} of {{total}}"
+msgstr ""
+
+msgid "Previous"
+msgstr ""
+
+msgid "Page"
+msgstr ""
+
+msgid "of {{pageCount}}"
+msgstr ""
+
+msgid "Next"
+msgstr ""
```

Comment:
We should create a follow up issue for the translations here that are part of a single sentence, but split up into separate translations. I don't think that this will lend itself well to translation to every language.

Reply:
https://jira.dhis2.org/browse/DHIS2-10348

Reply:
Thanks!